src/service/performance-impl.js

```diff
@@ -232,21 +232,26 @@ export class Performance {
    * @param {?string=} opt_from The label of a previous tick to use as a
    *    relative start for this tick.
    * @param {number=} opt_value The time to record the tick at. Optional, if
-   *    not provided, use the current time. You probably want to use
-   *    `tickDelta` instead.
+   *    not provided, use the current time.
+   * @param {number=} tickDelta The delta. Call tickDelta instead of setting
+   *     this directly.
    */
-  tick(label, opt_from, opt_value) {
+  tick(label, opt_from, opt_value, opt_delta) {
```

Comment:
In this case we can, because no other binaries use it. Otherwise we'd need to be careful.